src/utils/applescript.js

```
@@ -0,0 +1,361 @@
+        const maxLength = 200;
+        const logString = result.length > maxLength
+          ? result.substring(0, maxLength) + '...'
+          : result;
+        console.log('JSON string:', logString);
+
+        // 最後の手段として、手動でJSONを構築
+        try {
+          // 正規表現でイベント情報を抽出
+          const eventMatches = result.match(/Event: (.*?) at (.*?)$/gm);
+          if (eventMatches && eventMatches.length > 0) {
+            console.log(`Extracted ${eventMatches.length} events using regex`);
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)